store/event-store/src/test/java/org/calrissian/accumulorecipes/eventstore/impl/AccumuloEventStoreTest.java

```diff
@@ -136,6 +136,8 @@ public void testVisibility() {
         assertEquals(1, Iterables.get(actualEvent1, 1).size());
     }
 
+
+    @Ignore // ignoring for now. We may not need tuples to expire
```

Comment:
I want to think about the expiration functionality a lot more and the effect that it's going to have on the final documents. It adds a significant overhead to the deserializing of both the final rows and the metadata and I don't think it's something that should be enabled by default. I thought we talked about this offline- it's really not something that's used and it adds so much overhead- so I'd like to can it for now. It's easy enough to add back later if we need it.